### ISSUE TITLE
Mismatch between code and documentation of openpgp.decryptKey()

### ISSUE THREAD

**Issue by BAKFR** [openpgpjs/openpgpjs#473]
The documentation indicates that `decryptKey()` takes two parameters: the private key and the passphrase.
In the code, it takes only one "option" parameter, which must contains the key and the passphrase.

from https://openpgpjs.org/openpgpjs/doc/module-openpgp.html#.decryptKey

> (static) decryptKey(privateKey, passphrase) → {Key}

from openpgp.js:

``` javascript
/**
 * Unlock a private key with your passphrase.
 * @param  {Key} privateKey      the private key that is to be decrypted
 * @param  {String} passphrase   the user's passphrase chosen during key generation
 * @return {Key}                 the unlocked private key
 */
export function decryptKey({ privateKey, passphrase }) {
```

EDIT:
Also, the function returns a `Promise<{key: Key}>`, and not a `Key`.

**Comment by guitarmanusa** [comment]
I ran into this issue just yesterday as well.  When passing two arguments I received the error "Error decrypting private key: Cannot read property 'decrypt' of undefined".  

What worked was setting:
options = { privateKey: openpgp.key.readArmored(privkey), passphrase: 'password' };
decrypted_privkey = openpgp.decryptKey(options)._results.key;

The same thing also holds true for the decryptSessionKey() function
options = { message: openpgp.message.readArmored(text), privateKey: decrypted_privkey };
sessionKey = openpgp.decryptSessionKey(options)._result.data;

**Comment by twiss** [comment]
This has been fixed in e2eadd09e41242422d71fe526c989c5e5a42553c :blush:
